cli/command/system/version.go

```diff
@@ -48,16 +52,22 @@ Server:{{if ne .Platform.Name ""}} {{.Platform.Name}}{{end}}
    {{- end}}
   {{- end}}
  {{- end}}
-{{- end}}{{end}}`
+ {{- end}}{{- end}}
+ {{- if .KubernetesOK}}{{with .Kubernetes}}
+ Orchestrator:
+   Kubernetes:	{{.Kubernetes}}
+   Stack:	{{.StackAPI}}{{- end}}{{end}}`
```

Comment:
What if there is multiple API version available ? Should we display sthg a bit like docker api versions (version negociated vs min version) or a list of version, of just the version used ?

Reply:
I don't see the benefits of listing all the versions? At least with just the version used, there is no ambiguity.